### DISCUSSION TITLE
Discord Bot (Message Management)

### DISCUSSION THREAD

**Discussion by PR3SIDENT** [gameserverapp/Platform#542] [Suggestions]
When using the UI for GSA, I am confused on what "Settings" under "Integration Mod" means. Seemingly it appears to power both the Discord Bot's message, and potentially message In-Game via the same UI options. I believe a clearer or streamlined UI should exist under Configure > Discord Bot.

This is where administrators can change the values that the Discord Bot sends to the linked Discord servers.

I've attached a screenshot of the 3 suggested changes below.

1. Rename "Settings" under Integration Mod to "Broadcast Settings"
2. Expand options under "Configure" > "Discord Bot"
3. Allow custom values to exist for Discord communications/messages.

![Suggested Changes - Discord Bot](https://github.com/gameserverapp/Platform/assets/11300559/09b1ce54-3279-42ec-a4f0-9422f8bf75b8)

Thanks!

**Comment by mvaessen** [comment]
Since the messages on the Settings page are not just used for Discord, am I going to reject this one
